pagopa/io-app
# feat(Firma con IO): [SFEQS-1326] Update copy data sharing screen

## Short description
This PR updates copy

## List of changes proposed in this pull request
- Update locales

## How to test
Using io-dev-api-server start a signing flow and in `FciDataSharingScreen` new copy should appear correctly.
